code/modules/awaymissions/mission_code/snowdin.dm

```diff
@@ -218,7 +218,7 @@
 	if(plasma_parts.len)
 		var/obj/item/bodypart/burn_limb = pick(plasma_parts) //using the above-mentioned list to get a choice of limbs
 		burn_human.emote("scream")
-		ADD_TRAIT(burn_limb, TRAIT_PLASMABURNT, src)
+		ADD_TRAIT(burn_limb, TRAIT_PLASMABURNT, name)
```

Comment:
Why use `name` as the source here instead of `REF(src)` or `burn_limb.name`?  Was that intentional?